recipe/spyder-menu.json

```diff
@@ -1,10 +1,10 @@
 {
     "$schema": "https://json-schema.org/draft-07/schema",
     "$id": "https://schemas.conda.io/menuinst-1.schema.json",
-    "menu_name": "spyder",
+    "menu_name": "{{ DISTRIBUTION_NAME }} spyder",
```

Comment:
I think this requires `menuinst` 2.0.2 to work, right? If that's the case, please bump its requirement in `meta.yaml`